### PR TITLE
VS Tooling Updates

### PR DESCRIPTION
**Pull request by jwallra**
Changes reflect updates to Easy Auth and some lingering V1 terminology removal

### CODE REVIEW

File: templates/ProjectTemplate/CSharp/1033/MobileServicesApplicationv1.0/App_Start/Startup.MobileApp.cs [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+using Microsoft.Azure.Mobile.Server.Authentication;
 using Microsoft.Azure.Mobile.Server.Config;
+using supporttestService.DataObjects;
+using supporttestService.Models;
 using Owin;
-using $safeprojectname$.DataObjects;
-using $safeprojectname$.Models;
 
-namespace $safeprojectname$
+namespace supporttestService
```
Comment (phvannor) [inline templates/ProjectTemplate/CSharp/1033/MobileServicesApplicationv1.0/App_Start/Startup.MobileApp.cs]: is this change intentional?  What is supporttest?

Comment (jwallra) [inline templates/ProjectTemplate/CSharp/1033/MobileServicesApplicationv1.0/App_Start/Startup.MobileApp.cs]: No.... this is a mistake.  I will fix this
